client/src/pages/Family/Family.jsx

```diff
@@ -87,7 +87,7 @@ const Family = () => {
             </div>
             <div className="carousel-info__quote-block">
               <div className="quotes">
-                <p className="quote-text">{people[index] && people[index].position === 'CEO' && t('family.quote-text-anna')}</p>
+                <p className="quote-text">{people[index] && people[index].position === 'CEO' && t('family.quote-text-anna')}{people[index] && people[index].position === 'PR & Communications Manager' && t('family.quote-text-ver')}</p>
```

Comment:
```suggestion
                <p className="quote-text">{people[index] && (people[index].position === 'CEO' && t('family.quote-text-anna')) || (people[index].position === 'PR & Communications Manager' && t('family.quote-text-ver'))}</p>
```

Reply:
Better to add `quote` attribute to the object and add translation there
https://github.com/atomspace/atomspace.od.ua/blob/d7c2e1994a86c6733efa2600d8340df5840d26c3/client/src/pages/Family/Family.jsx#L21